lib/slack/web/faraday/response/raise_error.rb

```diff
@@ -10,6 +10,11 @@ def on_complete(env)
 
             error_message =
               body['error'] || body['errors'].map { |message| message['error'] }.join(',')
+
+            if Slack::Web.config.verbose_errors && body['response_metadata']
```

Comment:
This is where I could not figure out how to access the client that was used to make the request. If there's a convenient way to do that then I can change this to pull the setting from the client.